### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by AstaRom**
Just cleaning up the code from extra spaces and tabstops,
If no such change is needed, simply reject the request :)

### CODE REVIEW

**Comment by awawa-dev** [comment]
Hi
It's OK but two spaces in Markdown syntax (*md* files) have special meaning (new line) and in this case they should not be removed.

**Comment by AstaRom** [comment]
And I always deleted them and line breaks did not go astray.
Now I looked at the [fork page](https://github.com/AstaRom/HyperSerialEsp8266) for the 'README.md' file, it is already with changes.
The text remained the same, with all the hyphenation.

Maybe I'm not looking there)))

**Comment by AstaRom** [comment]
Found one difference. Only in **line 67** there was a change (the image has shifted). The rest is ok)

**Comment by awawa-dev** [comment]
Look into new `Usage in HyperHDR` and especially in `Pinout` section.

**Comment by AstaRom** [comment]
I cancel the changes for MD files, leave the rest. Right?

**Comment by awawa-dev** [comment]
I have been fighting for a long time with "wild" spaces that are created during some operations in Visual Studio or Visual Code because if they are not used to format certain sections (there are such in HyperHDR) I treat it as a issue. However, this does not apply to Markdown files, I would prefer not to these file at all them because they were used there deliberately.

**Comment by AstaRom** [comment]
Reverted the 'README.md' change.
Updated the request.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020,2021,2022 awawa-dev
+Copyright (c) 2020,2021,2022,2023 awawa-dev
```
Comment (AstaRom) [inline LICENSE:3]: Although this line could be written like this:
`Copyright (c) 2020-2023 awawa-dev`

**Pull request merged**